src/python_testing/MinimalRepresentation.py

```diff
@@ -18,9 +18,9 @@
 from dataclasses import dataclass, field
 
 from chip.tlv import uint
-from conformance_support import ConformanceDecision
-from global_attribute_ids import GlobalAttributeIds
-from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main
+from testing_support.conformance import ConformanceDecision
```

Comment:
`testing_support` feels a bit global. This is specific to matter/chip so how about either `matter_testing_suport` or `chip_testing_support` ? I realinze that is a larger delta, however maybe it can be done with rg and sd, like `rg 'from testing_support' --files-with-matches | xargs sd 'from testing_support' 'from matter_testing_support'` and then with a few more manual edits.

Reply:
above was using:
   https://github.com/BurntSushi/ripgrep
   https://github.com/chmln/sd
   
I found these really useful for mass refactoring in the sdk

Reply:
Resolved.